### DISCUSSION TITLE
vitest support

### DISCUSSION THREAD

**Discussion by ianzone** [golevelup/nestjs#588] [Ideas]
vitest is an officially documented use case, https://docs.nestjs.com/recipes/swc#vitest
however, when i use createMock  from '@golevelup/ts-jest', error occurred,
```
ReferenceError: jest is not defined
 ❯ createRecursiveMockProxy node_modules/.pnpm/@golevelup+ts-jest@0.3.7/node_modules/@golevelup/ts-jest/src/mocks.ts:54:3
 ❯ Object.get node_modules/.pnpm/@golevelup+ts-jest@0.3.7/node_modules/@golevelup/ts-jest/src/mocks.ts:94:22
```

**Comment by WonderPanda** [comment]
I don't have any plans at the moment to build out vitest support. If someone from the community feels like tackling it I'd be fine to integrate it into the package
